### ISSUE TITLE
[To Do] Multilanguage cognitive model deployment errors

### ISSUE THREAD

**Issue by ryanisgrig** [microsoft/botframework-solutions#2783]
#### What project is affected?
ToDo Skill
#### What language is this in?
Deployment scripts
#### What happens?
There are various errors when deploying multiple languages.

* Token recognition errors (highlighted below)
* Chinese ToDo app does not deploy with the error `CanonicalForm cannot be null or empty.`

#### What are the steps to reproduce this issue?
Run the deploy scripts on a new ToDo Skill

#### What were you expecting to happen?
All language apps deploy successfully 

#### Can you share any logs, error output, etc.?
**German**
```
> Parsing ToDo LU file ...
line 367:15 token recognition error at: 'Ä'
line 381:19 token recognition error at: 'ä'
line 385:17 token recognition error at: 'ö'
line 399:16 token recognition error at: 'ä'
line 433:25 token recognition error at: 'ü'
line 447:20 token recognition error at: 'ü'
line 451:17 token recognition error at: 'ü'
line 453:16 token recognition error at: 'ü'
line 455:16 token recognition error at: 'ü'
line 455:20 token recognition error at: 'ä'
line 507:18 token recognition error at: 'ä'
line 525:19 token recognition error at: 'ü'
line 541:17 token recognition error at: 'ß'
line 545:16 token recognition error at: 'ü'
line 565:16 token recognition error at: 'ü'
line 565:17 token recognition error at: 'ß'
line 581:18 token recognition error at: 'ü'
line 602:14 token recognition error at: 'ä'
line 606:15 token recognition error at: 'ä'
```

**Italian**
```
> Parsing ToDo LU file ...
line 410:19 token recognition error at: 'è'
line 589:11 token recognition error at: '''
```

**French**
```
> Parsing ToDo LU file ...
line 355:29 token recognition error at: '’'
line 387:16 token recognition error at: 'é'
line 389:16 token recognition error at: 'é'
line 389:18 token recognition error at: 'é'
line 407:16 token recognition error at: 'ô'
line 411:18 token recognition error at: 'é'
line 413:17 token recognition error at: 'ï'
line 421:23 token recognition error at: 'â'
line 425:16 token recognition error at: 'é'
line 439:30 token recognition error at: 'é'
line 443:19 token recognition error at: 'à'
line 445:20 token recognition error at: 'à'
line 449:20 token recognition error at: 'é'
line 455:17 token recognition error at: 'è'
line 455:25 token recognition error at: 'é'
line 461:16 token recognition error at: 'é'
line 461:25 token recognition error at: 'à'
line 477:20 token recognition error at: 'é'
line 491:17 token recognition error at: 'î'
line 493:16 token recognition error at: 'é'
line 493:23 token recognition error at: 'à'
line 493:27 token recognition error at: 'ê'
line 495:17 token recognition error at: 'ê'
line 497:16 token recognition error at: 'â'
line 499:16 token recognition error at: 'â'
line 499:20 token recognition error at: 'à'
line 501:16 token recognition error at: 'â'
line 521:16 token recognition error at: 'ô'
line 521:24 token recognition error at: 'é'
line 525:16 token recognition error at: 'ô'
line 531:15 token recognition error at: '"'
line 531:24 token recognition error at: '"'
line 541:36 token recognition error at: 'é'
line 547:17 token recognition error at: 'è'
line 565:16 token recognition error at: 'â'
line 575:16 token recognition error at: 'é'
line 577:16 token recognition error at: 'é'
line 591:15 token recognition error at: 'é'
line 593:13 token recognition error at: 'è'
```

**Spanish**
```
> Parsing ToDo LU file ...
line 368:17 token recognition error at: 'á'
line 412:18 token recognition error at: 'é'
line 414:17 token recognition error at: 'í'
line 424:26 token recognition error at: 'á'
line 454:18 token recognition error at: 'ó'
line 464:37 token recognition error at: 'í'
line 466:18 token recognition error at: 'ó'
line 484:18 token recognition error at: 'ó'
line 500:23 token recognition error at: 'í'
line 530:19 token recognition error at: 'ó'
line 534:16 token recognition error at: 'á'
line 546:20 token recognition error at: 'ó'
line 556:17 token recognition error at: 'ú'
line 572:17 token recognition error at: 'ú'
line 574:20 token recognition error at: 'í'
line 594:15 token recognition error at: 'ó'
```

**Chinese**
```
> Parsing ToDo LU file ...
line 401:15 token recognition error at: '冰'
line 401:16 token recognition error at: '淇'
line 401:17 token recognition error at: '淋'
line 403:15 token recognition error at: '咖'
line 403:16 token recognition error at: '啡'
line 405:15 token recognition error at: '啤'
line 405:16 token recognition error at: '酒'
line 407:15 token recognition error at: '奶'
line 407:16 token recognition error at: '油'
line 409:15 token recognition error at: '奶'
line 409:16 token recognition error at: '酪'
line 411:15 token recognition error at: '意'
line 411:16 token recognition error at: '大'
line 411:17 token recognition error at: '利'
line 411:18 token recognition error at: '面'
line 413:15 token recognition error at: '木'
line 413:16 token recognition error at: '薯'
line 413:17 token recognition error at: '淀'
line 413:18 token recognition error at: '粉'
line 415:15 token recognition error at: '松'
line 415:16 token recognition error at: '饼'
line 415:17 token recognition error at: '粉'
line 417:15 token recognition error at: '果'
line 417:16 token recognition error at: '冻'
line 419:15 token recognition error at: '柠'
line 419:16 token recognition error at: '檬'
line 421:15 token recognition error at: '核'
line 421:16 token recognition error at: '仁'
line 421:17 token recognition error at: '巧'
line 421:18 token recognition error at: '克'
line 421:19 token recognition error at: '力'
line 421:20 token recognition error at: '饼'
line 423:15 token recognition error at: '橙'
line 423:16 token recognition error at: '汁'
line 425:15 token recognition error at: '水'
line 427:15 token recognition error at: '水'
line 427:16 token recognition error at: '果'
line 429:15 token recognition error at: '沙'
line 429:16 token recognition error at: '拉'
line 431:15 token recognition error at: '洋'
line 431:16 token recognition error at: '葱'
line 433:15 token recognition error at: '浆'
line 433:16 token recognition error at: '果'
line 435:15 token recognition error at: '浓'
line 435:16 token recognition error at: '奶'
line 435:17 token recognition error at: '油'
line 437:15 token recognition error at: '火'
line 437:16 token recognition error at: '腿'
line 439:15 token recognition error at: '烟'
line 439:16 token recognition error at: '熏'
line 439:17 token recognition error at: '火'
line 439:18 token recognition error at: '鸡'
line 442:15 token recognition error at: '焦'
line 442:16 token recognition error at: '糖'
line 444:15 token recognition error at: '焦'
line 444:16 token recognition error at: '糖'
line 444:17 token recognition error at: '奶'
line 444:18 token recognition error at: '酪'
line 444:19 token recognition error at: '爆'
line 444:20 token recognition error at: '米'
line 444:21 token recognition error at: '花'
line 446:15 token recognition error at: '煎'
line 446:16 token recognition error at: '饼'
line 446:17 token recognition error at: '粉'
line 448:15 token recognition error at: '爆'
line 448:16 token recognition error at: '米'
line 448:17 token recognition error at: '花'
line 450:15 token recognition error at: '牛'
line 450:16 token recognition error at: '奶'
line 452:15 token recognition error at: '牛'
line 452:16 token recognition error at: '排'
line 454:15 token recognition error at: '狗'
line 454:16 token recognition error at: '粮'
line 456:15 token recognition error at: '猪'
line 456:16 token recognition error at: '排'
line 458:15 token recognition error at: '甜'
line 458:16 token recognition error at: '瓜'
line 458:17 token recognition error at: '糖'
line 460:15 token recognition error at: '白'
line 460:16 token recognition error at: '胡'
line 460:17 token recognition error at: '桃'
line 460:18 token recognition error at: '泥'
line 462:15 token recognition error at: '红'
line 462:16 token recognition error at: '糖'
line 464:15 token recognition error at: '芝'
line 464:16 token recognition error at: '士'
line 466:15 token recognition error at: '花'
line 466:16 token recognition error at: '生'
line 468:15 token recognition error at: '花'
line 468:16 token recognition error at: '生'
line 468:17 token recognition error at: '酱'
line 470:15 token recognition error at: '苹'
line 470:16 token recognition error at: '果'
line 472:15 token recognition error at: '苹'
line 472:16 token recognition error at: '果'
line 472:17 token recognition error at: '酱'
line 474:15 token recognition error at: '菠'
line 474:16 token recognition error at: '菜'
line 476:15 token recognition error at: '葡'
line 476:16 token recognition error at: '萄'
line 476:17 token recognition error at: '干'
line 478:15 token recognition error at: '蔬'
line 478:16 token recognition error at: '菜'
line 480:15 token recognition error at: '蔬'
line 480:16 token recognition error at: '菜'
line 480:17 token recognition error at: '沙'
line 480:18 token recognition error at: '拉'
line 482:15 token recognition error at: '西'
line 482:16 token recognition error at: '红'
line 482:17 token recognition error at: '柿'
line 484:15 token recognition error at: '醋'
line 486:15 token recognition error at: '面'
line 486:16 token recognition error at: '包'
line 488:15 token recognition error at: '饮'
line 488:16 token recognition error at: '料'
line 490:15 token recognition error at: '饼'
line 490:16 token recognition error at: '干'
line 492:15 token recognition error at: '香'
line 492:16 token recognition error at: '蕉'
line 494:15 token recognition error at: '鱼'
line 496:15 token recognition error at: '鸡'
line 496:16 token recognition error at: '蛋'
line 498:15 token recognition error at: '食'
line 498:16 token recognition error at: '物'
line 500:15 token recognition error at: '玉'
line 500:16 token recognition error at: '米'
line 502:15 token recognition error at: '酸'
line 502:16 token recognition error at: '奶'
line 504:15 token recognition error at: '巧'
line 504:16 token recognition error at: '克'
line 504:17 token recognition error at: '力'
line 506:15 token recognition error at: '蜂'
line 506:16 token recognition error at: '蜜'
line 508:15 token recognition error at: '蛋'
line 508:16 token recognition error at: '糕'
line 510:15 token recognition error at: '果'
line 510:16 token recognition error at: '汁'
line 512:15 token recognition error at: '胡'
line 512:16 token recognition error at: '萝'
line 512:17 token recognition error at: '卜'
line 514:15 token recognition error at: '果'
line 514:16 token recognition error at: '酱'
line 516:15 token recognition error at: '坚'
line 516:16 token recognition error at: '果'
line 518:15 token recognition error at: '糖'
line 518:16 token recognition error at: '果'
line 520:15 token recognition error at: '葡'
line 520:16 token recognition error at: '萄'
line 522:15 token recognition error at: '牛'
line 522:16 token recognition error at: '肉'
line 524:15 token recognition error at: '核'
line 524:16 token recognition error at: '桃'
line 526:15 token recognition error at: '土'
line 526:16 token recognition error at: '豆'
line 528:15 token recognition error at: '草'
line 528:16 token recognition error at: '莓'
line 530:15 token recognition error at: '糖'
line 532:15 token recognition error at: '糕'
line 532:16 token recognition error at: '点'
line 534:15 token recognition error at: '黄'
line 534:16 token recognition error at: '油'
line 536:15 token recognition error at: '盐'
line 540:10 token recognition error at: '购'
line 540:11 token recognition error at: '买'
line 543:10 token recognition error at: '买'
```

**Comment by bobokids** [comment]
Same as #2785

**Comment by ssss141414** [comment]
Could you please provide more information? Such as the version of your ludown, bf cli, deploy.ps1 or parameters in your deployment commands?

I have test deployment on **ludown 1.4.0** and **bf cli 4.7.0**. And I use **deploy.ps1** on branch **master** and **next**. All these deployments are OK.

Here is my process:

1 ludown and bf cli
![image](https://user-images.githubusercontent.com/18118994/71322690-6226de00-2505-11ea-97c6-ce3154a75081.png)

2 command
.\Deployment\Scripts\deploy.ps1 -name {name} -resourceGroup {resourceGroupName} -location westus -appPassword {appPassword} -luisAuthoringKey {luisAuthoringKey} -luisAuthoringRegion {luisAuthoringRegion}

3 result
On **master**:
![image](https://user-images.githubusercontent.com/18118994/71322749-fbee8b00-2505-11ea-957e-bc8705eba875.png)

On **next**:
![image](https://user-images.githubusercontent.com/18118994/71322774-4112bd00-2506-11ea-8c1d-3ca331896789.png)

Thanks!

**Comment by ssss141414** [comment]
Fixed in bf cli 4.7. Closed.
